src/lib.rs

```diff
@@ -132,8 +186,26 @@ fn fmt_lines(changes: &mut ChangeSet) {
         let mut line_len = 0;
         let mut cur_line = 1;
         let mut newline_count = 0;
+        let mut errors = vec![];
+        let mut issue_seeker = UnnumberedIssueSeeker::new();
+
         for (c, b) in text.chars() {
             if c == '\r' { continue; }
+
+            if config!(check_unnumbered_issues) {
+                // Add warnings for unnumbered todos/ fixmes
+                if let Some(issue_type) = issue_seeker.inspect(c) {
+                    let error_kind = match issue_type {
+                        IssueType::Todo => ErrorKind::UnnumberedTodo,
```

Comment:
Could inspect return an ErrorKind? Or perhaps you could have one `unnumbered` error kind rather than two, and take the IssueType as a field of the variant?

This match makes me sad for reasons I can't articulate very well